Source/DafnyRuntime/DafnyRuntimeRust/Cargo.toml

```diff
@@ -7,3 +7,6 @@ edition = "2021"
 once_cell = "1.18.0"
 num = "0.4"
 itertools = "0.11.0"
+
+[features]
+sync = []
```

Comment:
Worth a comment, since consumers have to turn on this feature and should be aware of what it's doing?